### PR TITLE
add custom map

### LINKED ISSUE #112: Map view: custom imagery/TMS
A number of field mapping projects use custom imagery and/or TMS services. Would be very useful for logistics, team/task distribution etc if these could be accessed from the main map view (I know FieldPapers had this functionality before)

### PR DESCRIPTION
**Pull request by ann26**
fix #112

### CODE REVIEW

**Comment by dimasciput** [comment]
@ann26 could you upload a gif or screenshots?

**Comment by ann26** [comment]
@dimasciput here is the gif
peek 2017-06-19 15-41

**Comment by cchristelis** [comment]
Hi @ann26, changing the map background should be a map attribute. I think we should rather have a control button on the map.

**Comment by ann26** [comment]
@cchristelis ok, done.
peek 2017-06-19 19-45

**Comment by cchristelis** [comment]
thanks

File: flask_project/campaign_manager/templates/campaign_detail.html [hunk 1]
```diff
@@ -193,6 +207,28 @@ <h3 class="detail-campaign-name">{{ name }}</h3>
             })
         }
 
+        function findAttribution(map_url){
```
Comment (meomancer) [inline flask_project/campaign_manager/templates/campaign_detail.html]: hi @ann26 
i think this list should be done on backend
create list of map and it's attribution
and return to the frontend
custom_map = { url: <url>, attribution:<attribution> }

**Comment by dimasciput** [comment]
Hi @ann26, I can't go back to custom after switch to default 

wizard

**Comment by ann26** [comment]
Hi @dimasciput , yeah you can't, should I make it so that it can go back to custom after you switch to default?

**Comment by dimasciput** [comment]
@ann26 yes please

**Comment by ann26** [comment]
@dimasciput ok, now you can switch back to custom
peek 2017-06-21 10-46

**Pull request merged**